ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -83,7 +83,10 @@ ghcLibHsSrcDirs :: [Cabal] -> [FilePath]
 ghcLibHsSrcDirs lib =
   let all = Set.fromList $
         [ "ghc-lib/stage1/compiler/build"
-        , "ghc-lib/stage0/libraries/ghci/build"
+        -- , "ghc-lib/stage0/libraries/ghci/build"
+        -- No, don't include this. Have Cabal generate 'InfoTable.hs'
```

Comment:
FWIW, those comments are somewhat inconsistent. In the Bazel case, Cabal does not come into play.